topic_benchmark/cli.py

```diff
@@ -22,11 +24,23 @@ def run_cli(
     encoder_model: str = "all-MiniLM-L6-v2", out_path: Optional[str] = None
 ):
     vectorizer = default_vectorizer()
+
     print("Loading Encoder.")
-    encoder = SentenceTransformer(encoder_model)
+    # first try to load encoder from registry
+    # if not found, assume encoder is a sentence transformer
+    try:
+        encoder = encoder_registry.get(encoder_model)
+    except:
```

Comment:
Bare except is not a good idea. There is info on why [here](https://stackoverflow.com/questions/54948548/what-is-wrong-with-using-a-bare-except). (For instance it would catch a `KeyboardInterrupt`, which we don't want)
Also I think this wouldn't fail here, because `.get()` might just return `None` if it is not present as far as I know.
The more specific we are with the `except` clause the better. In this case my guess is that it's a `KeyError`.